60.md

```diff
@@ -0,0 +1,205 @@
+
+## Spending History Event
+Clients SHOULD publish `kind:7376` events to create a transaction history when their balance changes.
+
+```jsonc
+{
+    "kind": 7376,
+    "content": nip44_encrypt([
+        [ "direction", "in" ], // in = received, out = sent
+        [ "amount", "1", "sats" ],
+        [ "e", "<event-id-of-spent-token>", "<relay-hint>", "created" ],
+    ]),
+    "tags": [
+        [ "a", "37375:<pubkey>:my-wallet" ],
+    ]
+}
+```
+
+* `direction` - The direction of the transaction; `in` for received funds, `out` for sent funds.
+* `a` - The wallet the transaction is related to.
+
+Clients MUST add `e` tags to create references of destroyed and created token events along with the marker of the meaning of the tag:
+* `created` - A new token event was created.
+* `destroyed` - A token event was destroyed.
+* `redeemed` - A [[NIP-61]] nutzap was redeemed.
+
+All tags can be [[NIP-44]] encrypted. Clients SHOULD leave `e` tags with a `redeemed` marker unencrypted.
+
+Multiple `e` tags can be added to a `kind:7376` event.
+
+# Flow
+A client that wants to check for user's wallets information starts by fetching `kind:10019` events from the user's relays, if no event is found, it should fall back to using the user's [[NIP-65]] relays.
+
+## Fetch wallet and token list
+From those relays, the client should fetch wallet and token events.
+
+`"kinds": [37375, 7375], "authors": ["<my-pubkey>"]`
+
+## Fetch proofs
+While the client is fetching (and perhaps validating) proofs it can use the optional `balance` tag of the wallet event to display a estimate of the balance of the wallet.
+
+## Spending token
+If Alice spends 4 sats from this token event
+```jsonconc
+{
+    "kind": 7375,
+    "id": "event-id-1",
+    "content": nip44_encrypt({
+        "mint": "https://stablenut.umint.cash",
+        "proofs": [
+            { "id": "1", "amount": 1 },
+            { "id": "2", "amount": 2 },
+            { "id": "3", "amount": 4 },
+            { "id": "4", "amount": 8 },
+        ]
+    }),
+    "tags": [
+        [ "a", "37375:<pubkey>:my-wallet" ]
+    ]
+}
+```
+
+Her client:
+* MUST roll over the unspent proofs:
+```jsonconc
+{
+    "kind": 7375,
+    "id": "event-id-2",
+    "content": nip44_encrypt({
+        "mint": "https://stablenut.umint.cash",
+        "proofs": [
+            { "id": "1", "amount": 1 },
+            { "id": "2", "amount": 2 },
+            { "id": "8", "amount": 8 },
+        ]
+    }),
+    "tags": [
+        [ "a", "37375:<pubkey>:my-wallet" ]
+    ]
+}
+```
+* MUST delete event `event-id-1`
+* SHOULD create a `kind:7376` event to record the spend
+```jsonconc
+{
+    "kind": 7376,
+    "content": nip44_encrypt([
+        [ "direction", "out" ],
+        [ "amount", "4", "sats" ],
```

Comment:
Like above, "sat" is consistent with the units of the mint.
```suggestion
        [ "amount", "4", "sat" ],
```

Reply:
my code is full of `s/sats/sat/g` -- this is a welcome addition 😂